### PR TITLE
relnote(117): Inline SVG allow for defer, async and module scripts

### PR DESCRIPTION
**Pull request by bsmth**
SVG2 should [treat `<script>`s as in HTML](https://svgwg.org/svg2-draft/interact.html#ScriptElement) although there's a few minor differences (`src` -> `href` attrs).

Because we're assuming functionality from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script, we should be able to use `defer`, `module`, and `async`. This PR adds compat info for these three under SVG.

__Open question:__
Should we be duplicating documenting attrs under SVG also, the differences only, or delegating to `<script>`?

__demo:__
https://github.com/bsmth/interactive-svg-demo

__WPT:__
* https://wpt.fyi/results/svg/interact/scripted?label=master&product=firefox%5Bexperimental%5D&product=chrome&product=safari&product=firefox%5Bstable%5D&aligned

__Related issues and pull requests:__
- [ ] Parent issue https://github.com/mdn/content/issues/28294

__Bugzilla:__
- [BUG-1839954](https://bugzilla.mozilla.org/show_bug.cgi?id=1839954)

### CODE REVIEW

File: svg/elements/script.json [hunk 1]
```diff
@@ -72,6 +136,38 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "module": {
+            "__compat": {
+              "support": {
```
Comment (Elchi3) [inline svg/elements/script.json:142]:
```suggestion
              "description": "<code>type='module'</code>",
              "support": {
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
We also need async and defer in the SVG DOM API:
https://github.com/mdn/browser-compat-data/blob/main/api/SVGScriptElement.json
https://hg.mozilla.org/mozilla-central/diff/21669d226d640b0b8d4b44449e512e0942b113a1/dom/webidl/SVGScriptElement.webidl

**Comment by bsmth** [comment]
> https://hg.mozilla.org/mozilla-central/diff/21669d226d640b0b8d4b44449e512e0942b113a1/dom/webidl/SVGScriptElement.webidl

Super, thanks for flagging

**Review by Elchi3 — COMMENTED** [review]
Thanks! Structurally it looks good to me now.

Regarding the versions:
- Firefox is set to `preview`, but in the referenced bugzilla bug it seems that it is shipping in 117? Do you have other information? If not, I would replace "preview" with "117".
- For Chrome, `false` is correct per https://bugs.chromium.org/p/chromium/issues/detail?id=1465850
- For Safari, `false` is correct per https://bugs.webkit.org/show_bug.cgi?id=75659

**Comment by Elchi3** [comment]
Oh and given Firefox is the only shipping engine, we need `experimental` set to to `true`.

**Comment by bsmth** [comment]
>Do you have other information? If not, I would replace "preview" with "117".

Yes, good call, I'm not sure why I set it to `preview` initially.

> we need `experimental` set to to `true`.

Great, `async` and `defer` done in 7c929c174ebf7592eb9087bbad61d9b8b94d34de and `module` in 10b880aeb306d9569e86387478e125e48c166db9

**Review by Elchi3 — APPROVED** [review]
Great work, merging 👍

**Comment by bsmth** [comment]
Thank you! 🙌🏻

**Pull request merged**